src/app/create-task/create-task.component.ts

```diff
@@ -33,22 +34,22 @@ export class CreateTaskComponent {
   }
 
   get title() {
-    return this.currentTask.get('title');
+    return this.currentTask.get('title')!;
```

Comment:
don't use non null assertion, I assume this is because when you pass it to the input validator component it throws an error, you should resolve the issue but not in this way 😄